internal/pkg/plugin/plugin.go

```diff
@@ -39,42 +56,3 @@ func loadPlugin(tool *config.Tool) DevStreamPlugin {
 
 	return devStreamPlugin
 }
-
-// Install loads the plugin and calls the Install method of that plugin.
-func Install(tool *config.Tool) {
```

Comment:
This is changed back to the original way of calling the plugin. It does not matter because this part will be changed by the "action/plan" part later.